### PR TITLE
Will Koster's Submission

### PR DESCRIPTION
**Pull request by ddsnowboard**

### CODE REVIEW

**Comment by zackpollard** [comment]
I don't think this is finished, just got a feeling

File: src/main/java/com/statefarm/codingcomp/agent/AgentParser.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 	@Cacheable(value = "agents")
 	public Agent parseAgent(String fileName) {
-		return null;
+            Agent out = new Agent();
+            out.
```
Comment (zackpollard) [inline src/main/java/com/statefarm/codingcomp/agent/AgentParser.java]: This line probably won't compile

**Comment by Vilsol** [comment]
Might be missing a few lines, but overall, looking good!

**Comment by aaomidi** [comment]
Looks finished to me tbh

**Comment by stuntguy3000** [comment]
Zack stop working for the enemy.

**Comment by ddsnowboard** [comment]
This is just so I don't forget at the end. I'm not actually done, believe it or not.

**Pull request closed without merging**